### ISSUE TITLE
Firmware Scan？

### ISSUE THREAD

**Issue by shadowhunter-YC** [google/osv-scanner#624]
Can OSV-SCANNER support firmware scan?  such as .bin, .image,. o ,. elf,.pkg 
I want to use it to test a firmware package for the Internet of Vehicles, such as TBOX

**Comment by andrewpollock** [comment]
No, these are not currently supported. Further to what you're trying to accomplish, do you have any examples of existing vulnerability records in OSV.dev that you'd hope to be able to find?
